index.html

```diff
@@ -6,173 +6,154 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>My Portfolio</title>
```

Comment:
- [ ] Kindly work on the border-radius and the margin of elements in this section to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/194722243-56c00121-fe44-413c-84dc-bb0235ef7fe4.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/194722379-a5a402d5-0a36-41e3-8996-2dfc88142d85.png)

- [ ] On your mobile version, Kindly try to align the elements to the center. You can give the overall container a padding.

![image](https://user-images.githubusercontent.com/98453979/194722456-4b6c51d5-5e10-4ecc-80c4-6f4ee0141ade.png)

### Checkout Fugma

![image](https://user-images.githubusercontent.com/98453979/194722491-07dacb09-2f62-4b64-bc8e-02b9c2e1c7ed.png)